src/main.scss

```diff
@@ -0,0 +1,71 @@
+$highlight: rgb(18, 18, 146);
+$errorHighlight: red;
+$textcolor: #333;
+$borderColor: #ccc;
+$labelColor: #aaa;
+$buttonColor: white;
+
+.font-loader {
+  font-family: 'Source Sans Pro', sans-serif;
+}
+
+.reuse  {
+ color: $highlight;
+}
+
+.form-group {
+	margin-top: 12px;
+  display: inline-block;
+  position: relative;
+}
+
+.input-field {
+  width: 250px;
+  padding: 4px 0;
+  font: 14px/24px "Lato", Arial, sans-serif;
+  color: $textcolor;
+  box-sizing: border-box;
+  letter-spacing: 1px;
+  border: 2px solid $borderColor;
+  border-bottom: 1px solid $borderColor;
+  background: transparent;
+}
+
+.label-text {
+  text-align: left;
+  position: absolute;
+  left: 0;
+  width: 100%;
+  top: 9px;
+  color: $labelColor;
+  z-index: -1;
+  letter-spacing: 0.5px;
+}
+
+.input-field:focus ~ .label-text
```

Comment:
btw do you know `~` is used here for?

Reply:
the bug of label sliding back to the original place is do the fact that styling is built on `:focus` state

Depending on `focus` state is fine, but you need to build a bit more logic around it